### PR TITLE
Fix pcre source and distfiles

### PR DESCRIPTION
**Pull request by craigcomstock**
Keep same version: pcre-8.44

Ticket: ENT-7927
Changelog: title

### CODE REVIEW

File: deps-packaging/pcre/source [hunk 1]
```diff
@@ -1 +1 @@
-https://ftp.pcre.org/pub/pcre/
+https://cfhcable.dl.sourceforge.net/project/pcre/pcre/8.44/
```
Comment (craigcomstock) [inline deps-packaging/pcre/source:1]: This URL is dumb. It is actually a redirect from the official URL, which may also work now that I have the checksum fixed.

**Comment by Lex-2008** [comment]
@cf-bottom will it build?

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7269/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7269/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7269/

**Comment by craigcomstock** [comment]
It did build! Only problem was that I put bad groovy post build script stuff in for fr-tests and sequential-tests. Fixed that and will re-run those two jobs.

**Comment by craigcomstock** [comment]
closing as I left a comment about what I think is a decent solution in @Lex-2008 's master deps update PR here: https://github.com/cfengine/buildscripts/pull/866

**Pull request closed without merging**